pkg/core/k8sobjects/resourcegroup.go

```diff
@@ -15,16 +15,31 @@
 package k8sobjects
 
 import (
+	kptv1alpha1 "github.com/GoogleContainerTools/config-sync/pkg/api/kpt.dev/v1alpha1"
 	"github.com/GoogleContainerTools/config-sync/pkg/core"
-	"github.com/GoogleContainerTools/config-sync/pkg/resourcegroup"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"github.com/GoogleContainerTools/config-sync/pkg/kinds"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// ResourceGroupObject initializes a ResourceGroup.
-func ResourceGroupObject(opts ...core.MetaMutator) *unstructured.Unstructured {
-	result := resourcegroup.Unstructured("", "", "")
-	defaultMutate(result)
+// ResourceGroupObjectV1Alpha1 initializes a kptv1alpha1.ResourceGroup.
+func ResourceGroupObjectV1Alpha1(ns, name string, opts ...core.MetaMutator) *kptv1alpha1.ResourceGroup {
```

Comment:
nit: `v1alpha1` is the only version currently so this could probably stay as `ResourceGroupObject`